docker/run-docker

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+####################################################################################
+#
+#   Run this file from the root of the repository to build and run a PWA container
+#
+####################################################################################
+
+ENVFILE=./.env
+HOSTSFILE=/etc/hosts
+HOSTPRESENT=false
+
+main () {
+  # copy environment variables to .env in root for docker-compose to consume for build
+  cp ./docker/.env.docker $ENVFILE 
```

Comment:
maybe not wipe out the current .env so that users can switch between both configurations

Reply:
see if this can be added to the docker-compose setup (adding an environment file for building) so that it doesn't require tampering with the host file system environment

Reply:
added a clean up of the .env at the end of this script and a message for users to switch to a new window if they want a clean environment to work outside of the docker setup